### ISSUE TITLE
Travis nightly (3.6) fails in coverage 3.7.1

### ISSUE THREAD

**Issue by jayvdb** [earwig/mwparserfromhell#112]
.travis.yml installs `coveralls`, which has a requirement coverage<3.999,>=3.6 , and it installs coverage 3.7.1.

The Travis target 'nightly' has switched to being Python 3.6, and coverage 3.7.1 fails on Python 3.6

```
$ coverage run --source=mwparserfromhell setup.py -q test
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6-dev/bin/coverage", line 9, in <module>
    load_entry_point('coverage==3.7.1', 'console_scripts', 'coverage')()
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/site-packages/pkg_resources/__init__.py", line 519, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2630, in load_entry_point
    return ep.load()
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2310, in load
    return self.resolve()
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2316, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/site-packages/coverage/__init__.py", line 65, in <module>
    use_cache = _singleton_method('use_cache')
  File "/home/travis/virtualenv/python3.6-dev/lib/python3.6/site-packages/coverage/__init__.py", line 46, in _singleton_method
    args, varargs, kw, defaults = inspect.getargspec(meth)
AttributeError: module 'inspect' has no attribute 'getargspec'
```

**Comment by earwig** [comment]
Python 3.6? What the hell?

**Comment by earwig** [comment]
@jayvdb I assume this is fixed by #113? I suppose 3.6 is still broken when running coveralls but mwparser has no reason to support 3.6 yet.

**Comment by jayvdb** [comment]
The immediate problem is fixed by #113 , but we should also report upstream the 3.6 problem in coveralls/coverage before closing this issue.

**Comment by earwig** [comment]
coverage.py seems to have moved on to version 4.0, which does not (as I can tell) use the problematic `inspect.getargspec`. coveralls claims to be in the process of adding support for it (coagulant/coveralls-python#47)? `pip install coveralls` still pulls 0.5, presumably because 1.0a2 from https://pypi.python.org/pypi/coveralls (which _should_ be using the newer version of coverage that _should_ be fine on Python 3.6) is still in alpha.

This is all very frustrating. Anyway, I think this issue will be fixed once a stable version of those packages is released.
